Dockerfile

```diff
@@ -1,14 +1,12 @@
-FROM --platform=$BUILDPLATFORM alpine AS builder
-
-FROM scratch
-COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
-
+FROM cgr.dev/chainguard/static:latest-20230102
```

Comment:
I'm not interested in using anything besides SCRATCH

Reply:
Essentially this is the scratch image that already has the ssl certs.

Reply:
I known (and trust) the Chainguard, but this provides no added value over the scratch image. The container is intentionally as lightweight as possible.